utils/weather.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request-promise');
+const ServerError = require('../errors/ServerError');
+
+const apiKey = require('config').weather.apiKey;
+const city = require('config').weather.city;
+const weatherUrl = `http://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}&units=metric`;
+
+class Weather {
+  async getWeather() {
+    try {
+      const options = {
+        uri: weatherUrl,
+      };
+      const response = await request(options);
+      const weather = JSON.parse(response);
```

Comment:
can we rename it ?  because weather.weather.maim and then. weather.main.temp confusing